### ISSUE TITLE
JSON bodies in tests 

### ISSUE THREAD

**Issue by dwalleck** [rackspace-titan/stacktester#24]
One thing I wanted to bring up was the design decision to include the JSON bodies statically within each test. I can see this causing severe headaches down the road as the specification changes. What I really liked about the test suite I had started with was the ability to have the format of a request abstracted away from the test itself so that when modifications are made that we only have one place to fix instead of many. The only solution I can think of based on the current framework would be to use helper methods to construct the body and pass it back to the test. An approach that would allow us to flip between using JSON and XML bodies also probably wouldn't be a bad idea.

**Comment by bcwaldon** [comment]
Personally, I think it would take a lot of work to get a useful set of xml testing working. We are also working on xml testing in nova's unit tests, so I'm not too worried about it at the moment. 

I also think there is value in having the bodies visible in the tests. That is a major part of what we are testing here.

**Comment by dwalleck** [comment]
True, but like with other code, I try to avoid repeating blocks of code in testing. I don't see how having the json body inside of the tests helps. I'm still on the fence about how I feel about testing xml only in unit testing, but as json is likely to be the most used format (I assume?), it's not as big of a concern.
